decidim-budgets/app/models/decidim/budgets/project.rb

```diff
@@ -36,6 +37,11 @@ def comments_have_votes?
       def confirmed_orders_count
         orders.finished.count
       end
+
+      # Public: Overrides the `notifiable?` Notifiable concern method.
+      def notifiable?(_context)
+        false
```

Comment:
Well.. I followed the same pattern as the `Commentable` module. I think it's just an issue about naming because the method is pretty handy to add some logic.